### PR TITLE
Cdk

### PR DESCRIPTION
**Pull request by schuettc**
**Issue #:** 

Replace SAM deployment with CDK deployment.  No other material changes.  

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
Deploying Demo

3. If you made changes to the component library, have you provided corresponding documentation changes?
N/A
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Confirmed and approved for MIT-0 as well.

### CODE REVIEW

**Comment by schuettc** [comment]
An error is generated by the React Hook "useMediaStreamMetrics"  Have removed it to complete testing but will need to be resolved.  

https://github.com/aws/amazon-chime-sdk-component-library-react/pull/509#issuecomment-851670337

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ verdaccio_app
 demo/*/yalc.lock
 demo/*/.yalc
 demo/chat/src/Config.js
+
+package-lock.json
```
[the remark this answers is not in the document]
Comment (schuettc) [reply .gitignore:15]: Sorry, this should be included.

File: demo/meeting/.gitignore [hunk 2]
```diff
@@ -1,2 +1,14 @@
-node_modules
-dist
+node_modules/
+package-lock.json
```
[the remark this answers is not in the document]
Comment (schuettc) [reply demo/meeting/.gitignore:2]: Yes, should be included.

File: demo/meeting/front-end-resources/front-end.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+        // Content bucket
+        const siteBucket = new s3.Bucket(this, 'SiteBucket', {
+            websiteIndexDocument: 'index.html',
+            websiteErrorDocument: 'index.html',
+            publicReadAccess: false,
+
+            // The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
+            // the new bucket, and it will remain in your account until manually deleted. By setting the policy to
+            // DESTROY, cdk destroy will attempt to delete the bucket, but will error if the bucket is not empty.
```
[the remark this answers is not in the document]
Comment (schuettc) [reply demo/meeting/front-end-resources/front-end.ts:29]: Yes, the S3 bucket will be emptied and then deleted.

File: demo/meeting/front-end-resources/react-meeting/src/constants/routes.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+const apiPath = cdkExports.MeetingBackEnd.apiURL
+
+const awsPath = '/prod';
+export const rootPath: string = window.location.href.includes(awsPath)
+  ? `${awsPath}/`
+  : '/';
+
+
+// export const apiPath = process.env.API_PATH
```
[the remark this answers is not in the document]
Comment (schuettc) [reply demo/meeting/front-end-resources/react-meeting/src/constants/routes.ts:14]: No.  That can be removed.

File: demo/meeting/front-end-resources/react-meeting/src/containers/Navigation/index.tsx [hunk 5]
```diff
@@ -38,7 +38,7 @@ const Navigation = () => {
         onClick={() => {}}
         label="Media metrics"
       >
-        <LocalMediaStreamMetrics />
+        {/* <LocalMediaStreamMetrics /> */}
```
[the remark this answers is not in the document]
Comment (schuettc) [reply demo/meeting/front-end-resources/react-meeting/src/containers/Navigation/index.tsx:41]: Yes.  Was running into an error from another commit.

https://github.com/aws/amazon-chime-sdk-component-library-react/pull/518#issuecomment-851670646

**Comment by schuettc** [comment]
Any updates?

**Comment by michhyun1** [comment]
Will close this for now - we've backlogged this item and we can re-open later if necessary.

**Pull request closed without merging**